src/PrestaShopBundle/Resources/views/Admin/Improve/International/Geolocation/geolocation.html.twig

```diff
@@ -0,0 +1,71 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain 'Admin.International.Feature' %}
+
+{% set geolocationByIpAddressForm, geolocationIpAddressWhitelistForm, geolocationOptionsForm = geolocationForm.geolocation_by_id_address, geolocationForm.geolocation_ip_address_whitelist, geolocationForm.geolocation_options %}
+
+{% block content %}
+  <div class="container" id="geolocationContent">
+    {% if not geolocationDatabaseAvailable %}
+    <div class="row">
+      <div class="col">
+        <div class="alert alert-warning" role="alert">
+          <span class="alert-text">
+            {{ 'In order to use Geolocation, please download [1]this file[/1] and extract it (using Winrar or Gzip) into the /app/Resources/geoip/ directory.'|trans({'[1]': '<a href="http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz" target="_blank">', '[/1]': '<a/>'})|raw }}
+          </span>
+        </div>
+      </div>
+    </div>
+    {% endif %}
+
+    {{ form_start(geolocationForm, {'action': path('admin_geolocation_process_form')}) }}
+    <div class="row">
+      <div class="col">
+        {% include '@PrestaShop/Admin/Improve/International/Geolocation/Blocks/geolocation_by_ip_address.html.twig' %}
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col">
+        {% include '@PrestaShop/Admin/Improve/International/Geolocation/Blocks/geolocation_options.html.twig' %}
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col">
+        {% include '@PrestaShop/Admin/Improve/International/Geolocation/Blocks/geolocation_ip_address_whitelist.html.twig' %}
+      </div>
+    </div>
+    {{ form_end(geolocationForm) }}
```

Comment:
missing form_rest in case we extend form

Reply:
```diff
- geolocation_form_rest
+ geolocation_form_reset
```